internal/pkg/utils/testhelper/runner/test.go

```diff
@@ -640,38 +642,6 @@ func (t *Test) assertProjectState() {
 	}
 }
 
-func waitForAPI(cmdErrCh <-chan error, apiURL string) error {
-	client := resty.New()
-
```

Comment:
Pre API E2E tu mame utilitku `waitForAPI`, spravil som z toho funkciu `testhelper.WaitForAPI` aby to slo pozit aj vo Worker E2E teste.